mars/deploy/oscar/tests/test_local.py

```diff
@@ -608,14 +608,15 @@ async def test_session_get_progress(create_cluster):
     assert session.session_id is not None
 
     raw = np.random.rand(100, 4)
-    t = mt.tensor(raw, chunk_size=5)
+    t = mt.tensor(raw, chunk_size=50)
```

Comment:
I don't think reducing num of chunks to 2 will be helpful for testing progresses.